src/library/library.cpp

```diff
@@ -421,3 +421,7 @@ void Library::slotSetTrackTableRowHeight(int rowHeight) {
     m_iTrackTableRowHeight = rowHeight;
     emit(setTrackTableRowHeight(rowHeight));
 }
+
+void Library::evictTrack(Track* pTrack) {
```

Comment:
is evict a common term for such function? I had to lookup the word.

Reply:
Knowing this from my CORBA days. It is a common term when dealing with caching: http://www.ehcache.org/documentation/2.7/apis/cache-eviction-algorithms.html